src/caret_analyze/architecture/architecture.py

```diff
@@ -719,12 +729,26 @@ def update_message_context(self, context_type: str,
                                    'subscription_topic_name': subscribe_topic_name,
                                    'publisher_topic_name': publish_topic_name})
 
-    def remove_callback_chain(self, subscribe_topic_name: str, publish_topic_name: str) -> None:
+    def remove_callback_chain(
+        self,
+        subscribe_topic_name: str,
+        subscription_construction_order: int | None,
```

Comment:
I think `None` is not necessary.

```suggestion
        subscription_construction_order: int,
```